corehq/apps/users/templates/users/partials/edit_role_modal.html

```diff
@@ -511,6 +511,64 @@ <h4 class="modal-title" data-bind="text: modalTitle"></h4>
             </div>
             {# END Roles & Permissions permissions #}
 
+            <legend>
+              {% trans "Reports" %}
+            </legend>
+
+            <div class="form-group">
+              <label class="col-sm-4 control-label">
+                {% trans "Create and Edit Reports" %}
```

Comment:
I've added "in report builder" to the label in the form-check (i.e. on the right) but left it out of the label on the form-group because it wrapped and looked out of place with the other labels in its column.